wger/core/templates/user/login.html

```diff
@@ -14,6 +14,18 @@
     {% trans 'Login' as submit_text %}
     {% render_form_fields form submit_text %}
 </form>
+  <br>
+  <p><strong>-- OR YOU CAN LOGIN WITH --</strong></p>
+  <a href="{% url 'social:begin' 'github' %}">GitHub</a> |
```

Comment:
***Code Quality: Nitpick***
We can have the social link as a context delivered to login screen and then loop through. that way it's easier to add more social link fro the view